metacat-main/src/main/java/com/netflix/metacat/main/services/notifications/sns/SNSNotificationServiceUtil.java

```diff
@@ -174,6 +179,25 @@ public TablePartitionsUpdatePayload createTablePartitionsUpdatePayload(
             .collect(Collectors.toList());
     }
 
+    /**
+     * get partition name list from list of partitionDtos. The returned list is capped at
+     * the first PARTS_UPDATED_LIST_MAX_SIZE elements, if there are more than that number of elements
+     * in the input then the return is empty which serves as a signal that complete list cannot be included
+     *
+     * @param partitionDtos partition DTOs
+     * @return list of partition ids from the input list
+     */
+    protected static List<String> getPartitionNameListFromDtos(final List<PartitionDto> partitionDtos) {
+        if (partitionDtos.size() > PARTS_UPDATED_LIST_MAX_SIZE) {
+            // empty list signals
+            return Collections.emptyList();
+        }
+        return partitionDtos.stream()
+            .map(PartitionDto::getName)
```

Comment:
Minor: We can as well make one mapper call with `dto -> dto.getName().getPartitionName()`

Reply:
Ok